docs/content/flow-port/staking-guide.mdx

```diff
@@ -30,6 +30,7 @@ You'll also need the following information about your node:
 - Network Address
```

Comment:
@JeffreyDoyle  A comment about some copy in Flow Port from the screenshots:

I feel like mentioning support for multiple machine account public keys will be more confusing than useful. It is supported by the transaction, but we only really support using one key for these accounts.